app/views/posts/_like_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="like-form">
+  <% if post.likes.find_by(user_id: current_user.id) %>
+    <%= link_to like_path(post.likes.find_by(user_id: current_user.id)), class: "like-button liked", method: :delete, remote: true do %>
+      <i class="glyphicon glyphicon-heart"></i>
+    <% end %>
+  <% else %>
+    <%= simple_form_for post.likes.build, remote: true do |f| %>
+      <%= f.input :likeable_type, as: :hidden, input_html: { value: 'Post' } %>
+      <%= f.input :likeable_id, as: :hidden, input_html: { value: post.id } %>
+      <%= button_tag type: 'submit', class: "like-button" do %>
+        <i class="glyphicon glyphicon-heart-empty"></i>
+      <% end %>
+    <% end %>
+  <% end %>
+</div>
+<% if post.likes.count > 0 %>
```

Comment:
add method in like model for example: 
```
def likeable_likes_count
  likeable.likes.count
end

Reply:
I don't know where I can use this method. In this case I can not use it, because I don't have **like** to call it (like.likeable_likes_count)

Reply:
Ok, I see. I thought that you use `like_form` partial only for likes#create and likes#destroy actions where you have @like. Feel free to remove this method.